src/index.html

```diff
@@ -1,18 +1,76 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <section
+      class="product-card"
+      data-qa="card"
+    >
```

Comment:
BEM methodology suggests avoiding setting several elements or blocks on the same DOM node. Here, `product-card` is a block, and it should not be mixed with an attribute like `data-qa`. It's better to place `data-qa="card"` on a child element inside the block, not on the block itself.